model/project_ref.go

```diff
@@ -447,6 +447,11 @@ func (p *ProjectRef) Add(creator *user.DBUser) error {
 		}
 	}
 
+	// Default to main if branch is empty.
+	if p.Branch == "" {
+		p.Branch = "main"
```

Comment:
Can we add a constant for this (maybe at the top of this file) just so we aren't hardcoding?

Reply:
:this: